lib/Service/ParticipantService.php

```diff
@@ -350,6 +371,36 @@ public function addUsers(Room $room, array $participants): void {
 		$this->dispatcher->dispatch(Room::EVENT_AFTER_USERS_ADD, $event);
 	}
 
+	public function getHighestPermissionAttendee(Room $room): ?Attendee {
+		$attendee = null;
+		try {
+			$roomOwners = $this->attendeeMapper->getActorsByParticipantTypes($room->getId(), [Participant::OWNER]);
+
+			if (!empty($roomOwners)) {
+				foreach ($roomOwners as $owner) {
+					if ($owner->getActorType() === Attendee::ACTOR_USERS) {
+						$attendee = $roomOwners[0];
```

Comment:
```suggestion
						$attendee = $owner;
```

Reply:
But what you actually want to do is:
```suggestion
						return $owner;
```